### DISCUSSION TITLE
https://penguins-eggs.net/blog/nessun-uomo-e-una-isola

### DISCUSSION THREAD

**Discussion by giscus** [pieroproietti/penguins-blog#4] [General]
# https://penguins-eggs.net/blog/nessun-uomo-e-una-isola

NESSUN UOMO È UN’ISOLA

https://penguins-eggs.net/blog/nessun-uomo-e-una-isola

**Comment by pieroproietti** [comment]
Si dice che nei momenti di sconforto - e ne ebbe - il povero John aggiungesse: salvo me!

**Comment by pieroproietti** [comment]
Questa è una mia libera traduzione di una poesia di John Donne [A Valediction: forbidding mourning](http://thewind61.blogspot.com/2009/03/un-invito-ad-evitare-il-pianto.html)
